src/main/kotlin/io/zerobase/smarttracing/resources/OrganizationsResource.kt

```diff
@@ -1,47 +1,150 @@
 package io.zerobase.smarttracing.resources
 
+import com.fasterxml.jackson.annotation.JsonProperty
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings
+import io.zerobase.smarttracing.GraphDao
+import io.zerobase.smarttracing.MultiMap
+import io.zerobase.smarttracing.models.OrganizationId
 import io.zerobase.smarttracing.models.IdWrapper
+import io.zerobase.smarttracing.models.SiteResponse
+import io.zerobase.smarttracing.models.ScannableId
 import javax.ws.rs.*
 import javax.ws.rs.core.MediaType
 
-//region Request Models
+/**
+ * Requests from clients.
+ */
+data class Contact(
+    @JsonProperty("phone") val phone: String,
+    @JsonProperty("email") val email: String,
+    @JsonProperty("contactName") val contactName: String
```

Comment:
`@JsonProperty` annotations are only required if the field name doesn't match the json field name. just remove all of these to reduce the noise